core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -471,6 +477,25 @@ public <T> ServerBuilder childChannelOption(ChannelOption<T> option, T value) {
         return this;
     }
 
+    /**
+     * (Advanced users only) Sets the {@link Consumer} that customizes the Netty {@link ChannelPipeline}.
```

Comment:
It does not override the previous one but appends it.
```suggestion
     * (Advanced users only) Adds the {@link Consumer} that customizes the Netty {@link ChannelPipeline}.
```